stingray/tests/test_lightcurve.py

```diff
@@ -83,29 +84,29 @@ def test_input_countrate(self):
         assert np.allclose(lc.counts, np.zeros_like(countrate) +
                            mean_counts*dt)
 
-    @raises(TypeError)
     def test_init_with_none_data(self):
```

Comment:
I think better is naming convention like:
`test_creating_litghtcurve_object_shoud_raises_type_error_when_initial_data_is_none`

2 arguments:
Lot of developers start reading new code from tests
Test report looks better, you don't need to go to code to see what you actually tested